### PR TITLE
Add follow/unfollow link to the contact actions menu

### LINKED ISSUE #10875: add Unfollow option to actions menu when on `View Contact`
Tested on Friendica, version 2021.12-dev that is running at the web location https://pirati.ca. The database version is 1438/1438, the post update version is 1427/1427.

Contact Actions menu:
contact   actions menu   no unfollow

Contact left side pane:
unfollow button in left side contact pane

Add `Unfollow` option to contact actions menu when on `View Contact` option. This is important for mobile. The way to unfollow a contact on mobile is to invoke the left side panel via the ellipsis icon. This works but has poor (none) visibility.

Edit: Split typo thing into separate issue https://github.com/friendica/friendica/issues/12112 and rephrased this issue here.

### PR DESCRIPTION
**Pull request by MrPetovan**
Closes #10875

### CODE REVIEW

File: src/Module/Contact/Profile.php [hunk 1]
```diff
@@ -415,6 +415,24 @@ private function getContactActions(array $contact, Entity\LocalRelationship $loc
 
 		$formSecurityToken = self::getFormSecurityToken('contact_action');
 
+		if ($localRelationship->rel & Contact::SHARING) {
```
Comment (annando) [inline src/Module/Contact/Profile.php:418]: You can unfollow a `FRIEND` as well.

File: view/theme/frio/templates/contact_edit.tpl [hunk 2]
```diff
@@ -19,6 +19,8 @@
 						</button>
 
 						<ul class="dropdown-menu pull-right" role="menu" aria-labelledby="contact-edit-actions-button" aria-haspopup="true" id="contact-actions-menu">
+                            {{if $contact_actions.follow}}<li role="presentation"><a role="menuitem" href="{{$contact_actions.follow.url}}" title="{{$contact_actions.follow.title}}">{{$contact_actions.follow.label}}</a></li>{{/if}}
```
Comment (annando) [inline view/theme/frio/templates/contact_edit.tpl]: Can you adjust the indentation?

File: view/theme/vier/templates/contact_edit.tpl [hunk 3]
```diff
@@ -16,8 +16,10 @@
 						<a class="btn" id="contact-edit-actions-button">{{$contact_action_button}}</a>
 
 						<ul role="menu" aria-haspopup="true" id="contact-actions-menu" class="menu-popup">
-							{{if $lblsuggest}}<li role="menuitem"><a href="#" title="{{$contact_actions.suggest.title}}" onclick="window.location.href='{{$contact_actions.suggest.url}}'; return false;">{{$contact_actions.suggest.label}}</a></li>{{/if}}
-							{{if $poll_enabled}}<li role="menuitem"><a href="#" title="{{$contact_actions.update.title}}" onclick="window.location.href='{{$contact_actions.update.url}}'; return false;">{{$contact_actions.update.label}}</a></li>{{/if}}
+                            {{if $contact_actions.follow}}<li role="menuitem"><a href="{{$contact_actions.follow.url}}" title="{{$contact_actions.follow.title}}">{{$contact_actions.follow.label}}</a></li>{{/if}}
```
Comment (annando) [inline view/theme/vier/templates/contact_edit.tpl]: Same like above :-)

File: src/Module/Contact/Profile.php (hunk 1 above)
Comment (MrPetovan) [reply src/Module/Contact/Profile.php:418]: Look at the operator, it's the binary AND, which means the following for relationships:
- None: 0 & 2 => false
- Follower: 1 & 2 => false
- Sharing: 2 & 2 => true
- Friend: 3 & 2 -> true

Comment (annando) [reply src/Module/Contact/Profile.php:418]: It's 0:30, my primary brain is already gone to sleep.

**Pull request merged**